src/components/ClassCards.scss

```diff
@@ -20,7 +21,7 @@
       grid-template-rows: repeat(3, max-content) 1fr max-content;
       border: 1px solid currentColor;
       border-radius: 2px;
-      background-color: var(--backgroundAlt);
+      background-color: var(--background);
```

Comment:
I'm suspicious of this change as the `backgroundAlt` was intended to make it white to pop off the page for the `/classes` page

background/context: this sort of cascading change on launch day for a simple design disagreement is why I didn't want to make a priority for today